### ISSUE TITLE
Load control

### ISSUE THREAD

**Issue by AntonioSun** [smicyk/groovy-jmeter#51]
https://github.com/smicyk/groovy-jmeter/blob/8e22a6230524704da27dae5aca5ebf686166ddf9/examples/commandline/script.groovy#L16

Questions

- How are the `users` `duration` variables passed? I see the calling script to be
`groovy script.groovy --jmx-out script.jmx -Vvar1=value1 -Vvar2=value2 -Vusers=1 -Vduration=10`
but I haven't find where the  `users` `duration` are specified. 

- To control the load, JMeter also has/need a third parameter --how long the script should run, or how many iterations. When such info is missing, how the user-load is exactly happening when groovy-jmeter scripts are being run?

Thanks!

**Comment by smicyk** [comment]
Hi,

the __users__ and __duration__ are created by groovy based on your command line input -Vusers=1 -Vduration=10. So you can create your own variable just by -Vmyvariablename=1 and then use it in the script as:
```
groupt(users: myvariablename) {
}
```

**Comment by AntonioSun** [comment]
Ah, yes, I missed that. thx!

**Comment by AntonioSun** [comment]
Also,

> To control the load, JMeter also has/need a third parameter --how long the script should run, or how many iterations. When such info is missing, how the user-load is exactly happening when groovy-jmeter scripts are being run?

how about this?

**Comment by AntonioSun** [comment]
Ah, gotcha. thx!

**Comment by AntonioSun** [comment]
> Or you can use **scheduler**, **delay** and **duration** to has more specific start time and end time.

I think the **delay** and **duration** might have somehow been switched. This is my source:

```
    group(name: 'Thread Group',
      users: '${c_lt_users}', rampUp: '${c_lt_ramp}',
      loops: '${c_lt_loops}', duration: 12000) {
```

and this is what I get from `--jmx-out`:

<img width="415" alt="image" src="https://user-images.githubusercontent.com/2840074/178012527-9975d8a3-bff6-434f-aa24-99f9710c5525.png">

BTW, while you're at it, I think the default name for Thread Group is "Thread Group", if you don't rename it after creating it within JMeter.  I think it'd better to use this as the default name, which will be acceptable to a broader audiences.

**Comment by smicyk** [comment]
Fixed with #96 

Regarding 'Thread Group' name. I made such convention to use User instead of Thread in the parameters names and in general when naming this concept. I think it makes more sense to use that name.
